src/controllers/appNotification.controller.js

```diff
@@ -0,0 +1,62 @@
+const httpStatus = require('http-status');
+
+const catchAsync = require('../utils/catchAsync');
+const {getPaginateConfig} = require('../utils/queryPHandler');
+const {appNotificationService, userService} = require('../services');
+const {userTypes} = require('../constants');
+const ApiError = require('../utils/ApiError');
+
+const createAppNotification = catchAsync(async (req, res) => {
+  const appNotification = await appNotificationService.createAppNotification({
+    ...req.body,
+    isCreatedByAdmin: true,
+  });
+  res.status(httpStatus.CREATED).json({data: appNotification});
+});
+
+const getAppNotifications = catchAsync(async (req, res) => {
+  const {options} = getPaginateConfig(req.query);
+  options.sortBy = 'createdAt';
```

Comment:
you can set these as default in validation schema

Reply:
Just saw that its already the default